.github/workflows/release-build-all.yml

```diff
@@ -23,11 +24,11 @@ jobs:
         run: |
           echo Git Base Ref: ${{ github.base_ref }}
           echo Git Ref Name: ${{ github.ref_name }}
-          echo Git Pull Run Number: ${{ github.run_number }}
-          echo Git Pull Request Ref: ${{ github.event.pull_request.head.sha }}
+          echo Git Head: ${{ github.event.pull_request.head.sha }}
+          echo Run Number: ${{ github.run_number }}
           echo OC CLI Version: $(oc version)
 
-  # Create new tag.
+  # # Create new tag.
```

Comment:
what happens with double `#`

Reply:
Leftover from when I was commenting and uncommenting the code. Just removed it.